packages/runtime/src/useCases/consumption/attributes/GetRepositoryAttributes.ts

```diff
@@ -1,12 +1,25 @@
 import { Result } from "@js-soft/ts-utils";
 import { AttributesController } from "@nmshd/consumption";
 import { Inject } from "typescript-ioc";
-import { AttributeMapper } from "..";
+import { AttributeMapper, GetAttributesRequestQuery, GetAttributesUseCase } from "..";
 import { LocalAttributeDTO } from "../../../types";
-import { SchemaRepository, SchemaValidator, UseCase } from "../../common";
+import { SchemaRepository, SchemaValidator, UseCase, flattenObject } from "../../common";
 
 export interface GetRepositoryAttributesRequest {
-    onlyLatestVersions?: boolean;
+    onlyLatestVersions?: boolean; // default: true
+    query?: GetRepositoryAttributesRequestQuery;
+}
+
+export interface GetRepositoryAttributesRequestQuery {
+    createdAt?: string;
+    "content.@type"?: string | string[];
+    "content.tags"?: string | string[];
+    "content.validFrom"?: string | string[];
+    "content.validTo"?: string | string[];
+    "content.key"?: string | string[];
```

Comment:
Afaik only RelationshipAttributes have a key, so this can be omitted here. Same for isTechnical and confidentiality.